README.md

```diff
@@ -8,6 +8,16 @@ The goal of this component is to collect and list all the deprecation of the pro
 
 Add `gem 'deprecations_detector'` to your application's Gemfile and execute `bundle`.
 
+Specify DeprecationDetector::Collect class as deprecation class with:
+
+```
+Rails.application.configure do
+  ...
+  config.active_support.deprecation = DeprecationsDetector::Collect
```

Comment:
Specifying this new class means that all deprecations are now sent to `DeprecationDetector` and not anymore to `stderr`?

Reply:
Yes, But as specified on the [PR description](https://github.com/nebulab/deprecations_detector/pull/8#issue-1830022358) this is a standard configuration of ActiveSupport.

You can send the warning to the stderr with the following configuration:

`config.active_support.deprecation = [:stderr, DeprecationsDetector::Collect]`

This is why we don't need the `suppress_deprecations` logic anymore

Reply:
Maybe it's better to specify it in the README as well. WDYT?